pkg/go/graph/weighted_graph.go

```diff
@@ -150,6 +150,21 @@ func (wg *WeightedAuthorizationModelGraph) AssignWeights() error {
 	return nil
 }
 
+// GetWeight returns the weight for the given key in the node, it could be a userset key or a terminal type key
```

Comment:
Can you call out what is the meaning of the boolean that is returned?  It is not obvious from the function name.